docs/library/micropython.rst

```diff
@@ -111,6 +111,13 @@ Functions
    Such an IRQ puts restrictions on the code that runs in the IRQ (for example
    the heap may be locked) and scheduling a function to call later will lift
    those restrictions.
+   
+   If the callback to be passed to ``schedule`` is a bound method, passing this
+   directly will fail. This is because creating a reference to a bound method
```

Comment:
"a bound method, passing this directly will fail." This isn't true in general. The paragraph apparently should start with "Note: If schedule is call from a preempting IRQ, when memory allocation is not allowed, ..."